tests/unit_tests/test_unit.py

```
@@ -649,47 +649,51 @@ def test_is_valid_url():
-    Test the calculation of amino acid and peptide scores from the raw amino
-    acid scores.
-    """
-    aa_scores_raw = np.asarray([0.0, 0.5, 1.0])
-
-    peptide_score = _peptide_score(aa_scores_raw, True)
-    assert peptide_score == pytest.approx(0.0)
-
-    peptide_score = _peptide_score(aa_scores_raw, False)
-    assert peptide_score == pytest.approx(-1.0)
+@pytest.mark.parametrize(
+    "aa_scores_raw, expected",
```

Ruff (0.15.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)